webrtc-stats.html

```diff
@@ -1100,7 +1100,10 @@
                 <dfn><code>remoteSource</code></dfn> of type <span class=
                 "idlMemberType"><a>boolean</a></span>
               </dt>
-              <dd></dd>
+              <dd>
+                True if the source is remote, for instance if it is sourced from another host via
```

Comment:
Just to confirm, does this mean that if a PeerConnection is re-transmitting a MediaStreamTrack received from *another* PeerConnection, `remoteSource` will be `true`? If so this sounds right.